extensions/amp-ad-network-fake-impl/0.1/data/fake_amp_ad_with_iframe_transport.html

```diff
@@ -1,20 +1,25 @@
 <!doctype html>
```

Comment:
Ignore changes to this file - it was changed in #10714 and will be rebased away.